st2common/st2common/hooks.py

```diff
@@ -30,6 +31,18 @@
 
 LOG = logging.getLogger(__name__)
 
+# A list of method names for which we don't want to log the result / response
+RESPONSE_LOGGING_METHOD_NAME_BLACKLIST = [
+    'get_all'
+]
+
+# A list of controller classes for which we don't want to log the result / response
+RESPONSE_LOGGING_CONTROLLER_NAME_BLACKLIST = [
```

Comment:
I know. I did add the blacklist last time either :P. In that case I noted it much later :)